### PR TITLE
Bug 1906866: Add translation for sample-utils.ts

### PR DESCRIPTION
**Pull request by vikram-raj**
**Fixes**
https://issues.redhat.com/browse/ODC-5231

**Description**
Add translation for sample-utils.ts

### CODE REVIEW

File: frontend/packages/console-shared/locales/en/console-shared.json [hunk 1]
```diff
@@ -75,5 +75,54 @@
...
+  "Deny": "Deny",
+  "all non-whitelisted traffic in the current namespace": "all non-whitelisted traffic in the current namespace",
+  "A fundamental policy by blocking all cross-pod traffics expect whitelisted ones through the other Network Policies being deployed.": "A fundamental policy by blocking all cross-pod traffics expect whitelisted ones through the other Network Policies being deployed.",
+  "traffic from external clients": "traffic from external clients",
+  "Allow external service from public Internet directly or through a Load Balancer to access the pod.": "Allow external service from public Internet directly or through a Load Balancer to access the pod.",
+  "traffic to an application from all namespaces": "traffic to an application from all namespaces",
+  "One typical use case is for a common database which is used by deployments in different namespaces.": "One typical use case is for a common database which is used by deployments in different namespaces.",
+  "traffic from all pods in a particular namespace": "traffic from all pods in a particular namespace",
```
[the remark this answers is not in the document]
Comment (vikram-raj) [reply frontend/packages/console-shared/locales/en/console-shared.json]: Yes, as Pod is a resource name.

**Comment by vikram-raj** [comment]
/retitle: Bug 1906866: Add translation for sample-utils.ts

**Review by jerolimov — APPROVED** [review]
Works fine :+1:

/lgtm

**Pull request merged**